### DISCUSSION TITLE
Add Taiwan currency

### DISCUSSION THREAD

**Discussion by gukii** [medusajs/medusa#3920] [Feature Requests]
Hi. The currency list is missing the "New Taiwan Dollar" currency, NTD for short.
However, the region Taiwan/China is present in the country list.

So, seems to just be missing the currency.

**Comment by olivermrbl** [comment]
Thanks for filing this @gukii. I'd be happy to accept a PR for it if you want to give it a go?

**Reply by gukii** [reply]
I ll give it a try. A PR from a fork, right?

**Reply by gukii** [reply]
The currency is already present. My bad..
